HyperwalletUISDK.xcodeproj/project.pbxproj

```diff
@@ -121,6 +121,15 @@
 		4A9E9A0F22F4AD5700739150 /* CreateTransferPresenterTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4A9A7BEC22E859E60022E573 /* CreateTransferPresenterTests.swift */; };
 		4A9E9A1022F4AD5700739150 /* ScheduleTransferPresenterTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 078C6EFB22EF7E0500CF81EE /* ScheduleTransferPresenterTests.swift */; };
 		4AB37CFC22EAF087005C1078 /* TransferAllFundsCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4AB37CFB22EAF087005C1078 /* TransferAllFundsCell.swift */; };
+		4AC464E7237F11EA00D7C092 /* HyperwalletInsightsMock.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4AF53728237DF03300C188FB /* HyperwalletInsightsMock.swift */; };
+		4AC464E8237F11EA00D7C092 /* HyperwalletInsightsMock.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4AF53728237DF03300C188FB /* HyperwalletInsightsMock.swift */; };
+		4AC464E9237F11EB00D7C092 /* HyperwalletInsightsMock.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4AF53728237DF03300C188FB /* HyperwalletInsightsMock.swift */; };
+		4AC464EA237F121700D7C092 /* AuthenticationProviderMock.swift in Sources */ = {isa = PBXBuildFile; fileRef = DB551FE622E1447B00DD65C2 /* AuthenticationProviderMock.swift */; };
```

Comment:
Please remove target membership of `AuthenticationProviderMock` from `Common` module
and `HyperwalletInsightsMock` from the `*RepositoryTests` modules

Reply:
fixed